cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -38,4 +38,10 @@
   public static final String SCHEDULE_NAME = "scheduleName";
```

Comment:
I think this constant was introduced in 2.7, and users have access to it. Is it okay to change it? It does not contain the namespace, which will always be the namespace of the application, yes.